### PR TITLE
Prow jobs and ci-operator config for knative-build-pipeline master...

### PR DESCRIPTION
**Pull request by mvinkler**
…branch

The image definitions and dockerfiles are prepared in this PR: [openshift/knative-build-pipeline#2](https://github.com/openshift/knative-build-pipeline/pull/2)

### CODE REVIEW

**Comment by droslean** [comment]
Please use the latest `ci-operator-prowgen` image to generate your pre/postsubmits

**Comment by mvinkler** [comment]
@droslean can you please check the PR again? I used latest `ci-operator-prowgen` image to generate pre/postsubmits, and all tests are passing now. Thank you.

File: ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  name: knative
+tests:
+- as: e2e
+  commands: "make test-e2e"
+  openshift_ansible_src:
+    cluster_profile: gcp
+resources:
+  '*':
+    limits:
+      cpu: '2'
```
Comment (droslean) [inline ci-operator/config/openshift/knative-build-pipeline/openshift-knative-build-pipeline-master.yaml]: please remove that.

**Comment by mvinkler** [comment]
@droslean removed cpu limit.

**Pull request merged**